synapse/app/homeserver.py

```diff
@@ -656,11 +658,13 @@ def start_generate_monthly_active_users():
         )
 
     start_generate_monthly_active_users()
```

Comment:
Don't you also want to move this into the if statement?

Reply:
Looks like it. 😄